vmdb/app/models/ems_amazon.rb

```diff
@@ -125,6 +125,20 @@ def vm_reboot_guest(vm, options = {})
     $log.error "MIQ(#{self.class.name}##{__method__}) vm=[#{vm.name}], error: #{err}"
   end
 
+  def stack_create(stack_name, template, options = {})
+    cloud_formation.stacks.create(stack_name, template.content, options).stack_id
+  rescue => err
+    $log.error "MIQ(#{self.class.name}##{__method__}) stack=[#{stack_name}], error: #{err}"
+    raise MiqException::MiqOrchestrationProvisionError, err.to_s
```

Comment:
You just want to re-raise the error here after logging so you can just call `raise` with no parameters.  Otherwise you will lose the original error type and stack trace.